rotkehlchen/chain/avalanche/manager.py

```diff
@@ -140,6 +145,10 @@ def get_transaction_receipt(
                 f'Couldnt deserialize transaction receipt '
                 f'data from covalent {tx_receipt} due to {str(e)}',
             ) from e
+        except KeyError as e:
+            raise RemoteError(
+                f'Couldn\'t find required data in the covalent\'s response. Missing key: {str(e)}',
```

Comment:
```suggestion
                f'Couldn\'t find required data in covalent\'s response. Missing key: {str(e)}',
```

But it's simpler to include it also in the above condition. And just change the {str(e)} to `msg` which you will conditionally check if key error. I told you to check how we do it in the rest of the codebase. Example: https://github.com/rotki/rotki/blob/develop/rotkehlchen/exchanges/ftx.py#L436